composer.json

```diff
@@ -131,7 +131,6 @@
         "monolog/monolog": ">=2",
         "phpdocumentor/reflection-docblock": "<3.0||>=3.2.0,<3.2.2",
         "phpdocumentor/type-resolver": "<0.3.0",
-        "ocramius/proxy-manager": "<2.1",
```

Comment:
Don't we need to keep this conflict rule?

Reply:
On 2nd though: you're right, since the proxy-manager is an optional dep :)

Reply:
I think we might even want to block all 2.x releases at the moment. Because you've removed the monkey-patching, we are not compatible to `ocramius/proxy-manager` anymore until your patch is merged upstream.

Reply:
That is not possible: the replace rule would conflict with the conflict rule you want me to add :)

Reply:
I see. 🙈

Reply:
I fixed the issue my moving "friendsofphp/proxy-manager-lts" to "require" instead of "require-dev" for "symfony/symfony".